cmd/recv.go

```diff
@@ -231,11 +231,13 @@ func recvAction(cmd *cobra.Command, args []string) {
 						bail("Error closing %s: %s", p, err)
 					}
 
-					rc.Close()
+					err = rc.Close()
+					if err != nil {
+						bail("Error closing %s: %s", p, err)
```

Comment:
In what case does this return an error that we need to care about?

Reply:
I don't know. It just feels bad to ignore errors. I'll revert this back.